dali/base/dasess.cpp

```diff
@@ -959,7 +942,16 @@ class CClientSessionManager: public CSessionManagerBase, implements IConnectionM
         udesc->serialize(mb);
         mb.append(auditflags);
         if (queryDaliServerVersion().compare("3.15") >= 0)
-            CDaliMessageSignatureHelper::serializeSignature(obj, udesc, mb);//serialize scope, signature, timestamp
+        {
+            CDateTime now;
```

Comment:
If you make the change in this PR you will always have a signature and a time passed to this function - it shouldn't be recalculating it.  The reqUTCTimestamp should also be a reference parameter.

Reply:
I don't follow what you are saying about recalculating ?

Reply:
I made the timestamp a reference type. In the PR, in workunit,cpp you will temporarily see me calling like this (adding the nullptr and "now"). This is temporary, in my PR for adding access token to workunits I have it being done correctly

CDateTime now;
perm = querySessionManager().getPermissionsLDAP("workunit",scopename,user,auditflags,nullptr,now);